src/gui/plugins/component_inspector/Pose3d.qml

```diff
@@ -185,14 +189,58 @@ Rectangle {
         // Left spacer
         Item {
           Layout.rowSpan: 3
-          width: margin + indentation
+          width: margin
```

Comment:
Can we keep the indentation? It helps communicate that the contents are iniside the header.